src/pyscaffold/utils.py

```diff
@@ -52,7 +52,7 @@ def chdir(path, **kwargs):
     try:
         with _chdir_logging_context(path, should_log):
             if not should_pretend:
-                os.chdir(path)
+                os.chdir(str(path))  # str to handle pathlib args
```

Comment:
I am under the impression that in Python 3, core `os` functions where updated to automatically handle `Path`-like objects in addition to strings. I was able to run the following with no erros:

```python
import os
from pathlib import Path
os.chdir(Path('/', 'tmp'))
```

Reply:
Interesting, for me it worked locally with Python 3.6 but it failed on Travis. Have you checked with Python 3.4? Maybe it was changed only in more recent versions.

Reply:
According to this [Stackoverflow question](https://stackoverflow.com/questions/41742317/how-can-i-change-directory-with-python-pathlib) it only works with Python 3.6 and above. I will add a ToDo that we drop it when we move to a required Python 3.6 version one day.